cmd/pebble/cmd_ls.go

```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/jessevdk/go-flags"
 
 	"github.com/canonical/pebble/client"
-	"github.com/canonical/pebble/internal/strutil/quantity"
+	"github.com/canonical/x-go/strutil/quantity"
```

Comment:
Nit: this should now go in the group of import lines with `jessevdk/go-flags`. We normally have three import groups: stdlib, external, and project-internal (`canonical/pebble`).

Same in other files.

Reply:
Fixed everywhere! Thank you.